core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -273,6 +273,8 @@ trait EntityComponent {
       }
     }
 
+    def refIdMap(entityRecords: Seq[EntityRecord]): Map[AttributeEntityReference, Long] = entityRecords.map(e => e.toReference -> e.id).toMap
```

Comment:
I am not convinced that extracting 1 liners into shared functions is worthwhile. Now, when I look at the code and see a call to refIdMap, I think "what is that, what is the key, what is the value?" and have to go and look instead of having it right there.

But that is just my opinion and I don't care enough to talk about it further. Ask around and do as you see fit.